src/mscorlib/src/System/Collections/Hashtable.cs

```diff
@@ -66,6 +66,7 @@ namespace System.Collections
     // 
     [DebuggerTypeProxy(typeof(System.Collections.Hashtable.HashtableDebugView))]
     [DebuggerDisplay("Count = {Count}")]
+    [Serializable]
```

Comment:
How is it possible that StringDictionary which lives in corefx uses the internal Hashtable from coreclr?
https://github.com/dotnet/corefx/blob/master/src/System.Collections.Specialized/src/System/Collections/Specialized/StringDictionary.cs#L21

```
  Deserialization of type System.Collections.Specialized.StringDictionary failed
     System.Runtime.Serialization.Formatters.Tests.BinaryFormatterTests.ValidateTfmHashes(original: [System.Collections.DictionaryEntry, System.Collections.DictionaryEntry], tfmBase64Hashes: [\"AAEAAAD/////AQAAAAAAAAAMAgAAAGFTeXN0ZW0uQ29sbGVjdG\"..., \"AAEAAAD/////AQAAAAAAAAAMAgAAAElTeXN0ZW0sIFZlcnNpb2\"...]) [FAIL]
        System.Runtime.Serialization.SerializationException : Type 'System.Collections.Hashtable' in Assembly 'System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e' is not marked as serializable.
```